main/test/ca/uwaterloo/flix/library/LibrarySuite.scala

```diff
@@ -35,14 +35,15 @@ class LibrarySuite extends Suites(
   new FlixTest("TestOption", "main/test/ca/uwaterloo/flix/library/TestOption.flix")(Options.TestWithLibrary),
   new FlixTest("TestRandom", "main/test/ca/uwaterloo/flix/library/TestRandom.flix")(Options.TestWithLibrary),
   new FlixTest("TestResult", "main/test/ca/uwaterloo/flix/library/TestResult.flix")(Options.TestWithLibrary),
-  new FlixTest("TestList", "main/test/ca/uwaterloo/flix/library/TestList.flix")(Options.TestWithLibrary),
+  new FlixTest("TestMutList", "main/test/ca/uwaterloo/flix/library/TestMutList.flix")(Options.TestWithLibrary),
```

Comment:
Did you accidentally remove TestList?

Reply:
Sure did! Sorry about that. I have restored it in the latest commit.